R/explanation.R

```diff
@@ -8,9 +8,9 @@
 #' @param explainer An \code{explainer} object to use for exaplaining the observations.
 #' See \code{\link{shapr}}.
 #'
-#' @param approach String or list. If a String, the approach to use when computing the Shapley
-#' values. Equal to 'empirical', 'gaussian' or 'copula'. If a list, the elements in the list
-#' refers to the rows in \code{x} that ought to be included in each of the approaches.
+#' @param approach Character. Note that \code{1 <= length(approach) <= n_features}, where
```

Comment:
Suggests:  @param approach Character (vector). Note that ....

Reply:
Fixed with dee3db7.